web_poet/testing/__main__.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+from pathlib import Path
+
+from web_poet.testing import Fixture
+
+
+def rerun(args):
+    fixture = Fixture(Path(args.fixture_path))
+    item = fixture.get_output()
+    if args.fields:
+        fields = args.fields.split(",")
+        item = {field: item[field] for field in fields}
```

Comment:
I think we can improve the error message here when users input a non-existing field. Or maybe ignore/supress the erorr?

Reply:
I was hoping nobody will notice it, and it'd be ok to just have a KeyError in the output :P

Reply:
Done in https://github.com/scrapinghub/web-poet/pull/132/commits/c2b18065bd7837f0c3ea27257084a8b511cbe241.